test/tools/perLayerAnalyzer/README.md

```diff
@@ -0,0 +1,145 @@
+```
+   {
+     "urlServer": "IP:8080",
+     "modelName": ["squeezenet1.1"],
+     "platform": "Mac",
+     "chromiumPath": "/Users/test/Downloads/Chromium.app/Contents/MacOS/Chromium",
+     "supportSwitch": false,
+     "API": "polyfill",
+     "preference": "fast"
+     "iterations": "200"
+   }
+```
+   or
+
+```
+   {
+      "urlServer": "IP:8080",
+      "modelName": ["squeezenet1.1"],
+      "platform": "Linux",
+      "chromiumPath": "/usr/bin/chromium-browser-unstable",
+      "supportSwitch": false,
+      "API": "polyfill",
+      "preference": "fast",
+      "iterations": "200"
+   }
+```
+   or
+
+```
+   {  
+     "urlServer": "IP:8080",
+     "modelName": ["squeezenet1.1"],
+     "platform": "Windows",
+     "chromiumPath": "C:\\test\\win_x64_SUCCEED\\Chrome-bin\\chrome.exe",
+     "API": "polyfill",
+     "preference": "fast",
+     "supportSwitch": false,
+     "iterations": "200"
+   }
+```
+   You need modify these eight fields for the different platforms:
+   + **_urlServer_**: `{string}`, Server IP address, port number 8080.
+   + **_modelName_**: `{array}`, We support **squeezenet1.1**, **mobilenetv2-1.0** two models, you need choose **["squeezenet1.1"]** , **["mobilenetv2-1.0"]** , **["squeezenet1.1", "mobilenetv2-1.0"]** , **["mobilenetv2-1.0", "squeezenet1.1"]** .
+   + **_platform_**: `{string}`, target platform, support **Android**, **Mac**, **Linux** and **Windows**.
+   + **_chromiumPath_**: `{string}`, **Mac**/**Linux**/**Windows**: the target chromium path **Android**: the chrome or chromium path in above Prerequisites to show the final checking results.
+   + **_supportSwitch_**: `{boolean}`, support **true** and **false**.
+   + **_API_**: `{string}`, choose to **polyfill** and **webnn**.
+   + **_preference_**: `{string}`, choose to  **fast** , **sustained** and **low**.
+   + **_iterations _**: `{string}`, set the number of times you want to run the realmodel case.
+
+|    |  Platform  |  Fast  |  Sustained  |  Low  |
+|  :-----:  |  :----:  |   :----:   |   :----:   |   :----:   |
+|  swith:true API:webnn  |  Windows   |    MKLDNN   |    DirectML   |   DirectML   |
+|  swith:true API:webnn  |  macOS  |  MKLDNN  |  MPS  |      |
+|  swith:true API:webnn  |  Linux  |   IE-MKLDNN	   |   IE-clDNN	   |   IE-MYRIAD  |
+|  swith:false API:webnn  |  Windows  |  MKLDNN  |  clDNN  |      |
+|  swith:false API:webnn  |  macOS  |   BNNS   |   MPS   |      |
+|  swith:false API:webnn  |  Linux   |    MKLDNN   |    clDNN   |      |
+|  swith:false API:polyfill  |  Windows  |   WASM   |   WebGL   |      |
+|  swith:false API:polyfill  |  macOS  |   WASM   |   WebGL   |      |
+|  swith:false API:polyfill  |  Linux  |   WASM   |   WebGL   |      |
+
+
+## Support Platforms
+
+|  Linux  |   Mac   |  Android  |  Windows  |
+|  :---:  |  :---:  |   :---:   |   :---:   |
+|  PASS   |   PASS  |    PASS   |    PASS   |
+
+
+## How TO Use Template.html File ( If you use one of the above ways, you don`t  need follow these steps )
```

Comment:
@shenx3x  There's a space to enhance generating automatically test html page by implementing with scripts, let's submit another PR for it later.